modules/Microsoft.ContainerInstance/containerGroups/.test/min/deploy.test.bicep

```diff
@@ -0,0 +1,64 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.containerinstance.containergroups-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'cicgmin'
+
+// =========== //
+// Deployments //
+// =========== //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '<<namePrefix>>${serviceShort}001'
+    lock: 'CanNotDelete'
```

Comment:
I'd suggest to only include required parameters in the min test. So to remove `lock` and `ipAddressPorts`.

Reply:
In the given configuration, the IPAddressPorts are actually required. However, I don't know the service well enough to judge if that makes it a generally required parameter. Lock is removed

Reply:
I'll then open an issue to investigate on it and approve meanwhile

Reply:
Created issue #2205.